File: examples/python/control_flow.py
total = 0
for index in range(1, 10):
    if index == 2:
        continue
    elif index == 5:
        break
    else:
        total = total + index
else:
    total = 999

count = 0
for value in range(3):
    count = count + 1
else:
    count = count + 10

completed = 0
while completed < 3:
    completed = completed + 1
else:
    completed = completed + 10

stopped = 0
while stopped < 3:
    stopped = stopped + 1
    break
else:
    stopped = 100

print(total, index, count, completed, stopped)

File: examples/fortran/control_flow.f90
program control_flow
  implicit none
  integer :: total = 0, index
  do index = 1, 10
    if (index .eq. 2) then
      cycle
    else if (index .eq. 5) then
      exit
    else
      total = total + index
    end if
  end do
  print *, total, index
end program control_flow
